src/bridge.js

```diff
@@ -16,8 +16,15 @@ import { processEvent } from './events';
 import EnplugError from './errors/EnplugError';
 
 
+
 // todo finish reject timeout
 const RESPONSE_TIMEOUT = (60 * 1000);
+const VERSION = '0.3.8-dev14';
```

Comment:
Consider importing package.json and using version property here.
Or don't and I'll do it in the refactor :)

Reply:
I'll leave it as is for now